src/overlays/actors/ovl_Demo_Tre_Lgt/z_demo_tre_lgt.h

```diff
@@ -6,11 +6,33 @@
 
 struct DemoTreLgt;
 
+typedef void (*DemoTreLgtActionFunc)(struct DemoTreLgt*, GlobalContext*);
+
+typedef struct {
+    /* 0x00 */ f32 startFrame;
+    /* 0x04 */ f32 endFrame;
+    /* 0x08 */ f32 unk_08;
+    /* 0x0C */ f32 unk_0C;
+} FrameInfo; // size = 0x10
+
+typedef struct {
+    /* 0x00 */ FrameInfo frameInfo[2];
+} DemoTreLgtInfo; // size = 0x20
+
 typedef struct DemoTreLgt {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x30];
+    /* 0x014C */ SkelAnimeCurve skelCurve;
+    /* 0x016C */ s32 action;
+    /* 0x0170 */ u32 unk_170; // some sort of alpha
+    /* 0x0174 */ u32 unk_174; // another sort of alpha
+    /* 0x0178 */ u8 status;
 } DemoTreLgt; // size = 0x017C
 
+typedef enum {
+    /* 0x00 */ DEMO_TRE_LGT_ACTION_WAIT, // wait until animation is needed
+    /* 0x01 */ DEMO_TRE_LGT_ACTION_ANIMATE,
```

Comment:
Trailing commas in enum trigger a warning
```suggestion
    /* 0x01 */ DEMO_TRE_LGT_ACTION_ANIMATE
```